Musee du Louvre.js

```diff
@@ -164,81 +164,56 @@ function doWeb(doc, url) {
 var testCases = [
 	{
 		"type": "web",
-		"url": "http://www.louvre.fr/llv/activite/detail_evenement.jsp?CONTENT%3C%3Ecnt_id=10134198673390879&CURRENT_LLV_ACTIVITE%3C%3Ecnt_id=10134198673390879&FOLDER%3C%3Efolder_id=9852723696500927",
+		"url": "http://www.louvre.fr/oeuvre-notices/stele-figurant-la-deesse-ishtar",
 		"items": [
 			{
 				"itemType": "artwork",
+				"title": "Stèle figurant la déesse Ishtar",
 				"creators": [],
-				"notes": [],
-				"tags": [
-					",Activité ",
-					"Musée du Louvre",
-					"Musée",
-					"Louvre "
-				],
-				"seeAlso": [],
-				"attachments": [],
-				"abstractNote": "Présentation centrée sur l'étude d'une oeuvre",
-				"title": "Détails des activités <i>La Dentellière</i> de Vermeer",
-				"extra": "Visites guidées",
-				"url": "http://www.louvre.fr/llv/activite/detail_evenement.jsp?CONTENT%3C%3Ecnt_id=10134198673390879&CURRENT_LLV_ACTIVITE%3C%3Ecnt_id=10134198673390879&FOLDER%3C%3Efolder_id=9852723696500927",
+				"abstractNote": "Stèle : déesse Ishtar . Provenance : Tell Ahmar, antique Til Barsip. Epoque néo-assyrienne, VIIIe siècle av. J.-C. - Département des Antiquités orientales",
```

Comment:
This comes directly from the corresponding `meta` tag:
```html
<meta name="description" content="Stèle : déesse Ishtar . Provenance : Tell Ahmar, antique Til Barsip. Epoque néo-assyrienne, VIIIe siècle av. J.-C. - Département des Antiquités orientales" />
```
Moreover, I am not sure how exactly the correct spacing would look like here and if there is a general pattern for that... I guess we could update and improve more things in this translator, but currently I wanted to concentrate that it works again. Can we maybe postpone this and make in issue for it to look at it later?